### PR TITLE
OMIS notification adviser added

### PR DESCRIPTION
**Pull request by marcofucci**
This sends a notification to OMIS advisers when they are added to an order.

### CODE REVIEW

File: datahub/omis/notification/test/test_templates.py [hunk 1]
```diff
@@ -68,3 +70,21 @@ def test_quote_awaiting_acceptance_for_contact(self, settings):
         order = OrderWithOpenQuoteFactory()
 
         notify.quote_generated(order)
+
+    def test_you_have_been_added_for_adviser(self, settings):
+        """
+        Test the notification for when an adviser is added to an order.
+        If the template variables have been changed in GOV.UK notifications this
+        is going to raise an exception.
```
Comment (elcct) [inline datahub/omis/notification/test/test_templates.py]: Do we know what type of exception?

File: datahub/omis/notification/signal_receivers.py [hunk 2]
```diff
@@ -17,3 +17,16 @@ def notify_post_save_order(sender, instance, created, **kwargs):
 def notify_post_quote_generated(sender, order, **kwargs):
     """Notify contact that a quote has been generated."""
     notify.quote_generated(order)
+
+
+@receiver(post_save, sender=OrderAssignee, dispatch_uid='notify_post_save_assignee')
+@receiver(post_save, sender=OrderSubscriber, dispatch_uid='notify_post_save_subscriber')
+def notify_post_save_order_adviser(sender, instance, created, **kwargs):
+    """Notify adviser that they have been added to the order"""
+    if created:
```
Comment (reupen) [inline datahub/omis/notification/signal_receivers.py]: Probably want to skip when raw=True?

Comment (marcofucci) [reply datahub/omis/notification/signal_receivers.py]: 🤔  never thought about raw=True. Is it only True when loading fixtures?

Comment (reupen) [reply datahub/omis/notification/signal_receivers.py]: Yes (it comes from `Model.save_base()`).

**Pull request merged**